features/google/src/model/google-account.js

```diff
@@ -0,0 +1,99 @@
+//TODO: Use DataAccount super class
+export default class GoogleAccount {
+    constructor() {
+        this._importingResults = [];
+        this._importedFileNames = [];
+
+        this._name = "";
+        this._preferredLanguage = [];
+        this._analyses = {};
+        this._reports = {};
+        this._analysesExecutionResults = [];
+        this._placeVisits = [];
+        this._activitySegments = [];
+        this._activities = [];
+        this._pathNames = [];
+    }
+
+    get analyses() {
+        return this._analyses;
+    }
+
+    get reports() {
+        return this._reports;
+    }
+
+    get importedFileNames() {
+        return this._importedFileNames;
+    }
+
+    addImportedFileName(fileName) {
+        this._importedFileNames.push(fileName);
+    }
+
+    get importingResults() {
+        return this._importingResults;
+    }
+
+    set importingResults(importingResults) {
+        this._importingResults = importingResults;
+    }
+
+    get analysesExecutionResults() {
+        return this._analysesExecutionResults;
+    }
+
+    set analysesExecutionResults(analysesExecutionResults) {
+        this._analysesExecutionResults = analysesExecutionResults;
+    }
+
+    get placeVisits() {
```

Comment:
Are we going to give every single feature an attribute? Wouldn't it be easier to just set them in runtime and generate accessors also in runtime?

Reply:
Hmm does give us an overview what we can expect and not let random attributes being set. That's why we went with this in fbImport